### DISCUSSION TITLE
Play Evolutions With MariaDB?

### DISCUSSION THREAD

**Discussion by Rapierian** [playframework/playframework#11654] [Q&A] [answered]
Hello, I'm attempting to use MariaDB with Play 2.8.19. So my database URL looks something like this:

DATABASE_URL: jdbc:mariadb://something.somethingelse.com...

This means that [this match statement in Application Evolutions](https://github.com/playframework/playframework/blob/a83c8cb39d95c3e31b03b01a693fd1a5c0d9f24a/persistence/play-jdbc-evolutions/src/main/scala/play/api/db/evolutions/ApplicationEvolutions.scala#L281) matches to the default case rather than the MysqlJdbcUrl case, which means that the syntax to create the lock table isn't using the right character for escape. 

Is there a way to tell play evolutions which lock syntax to use?

**Comment by mkurz** [comment]
@Rapierian This is hardcoded here:
https://github.com/playframework/playframework/blob/bcf814cd03753fbe75a746c9ca51b57bf72e6dfb/persistence/play-jdbc-evolutions/src/main/scala/play/api/db/evolutions/Evolutions.scala#L93
If we assume the mariadb syntax still is compatible with the mysql one, we can just change this regex ot `^(jdbc:)?(mysql|mariadb):.*` and your problem should be solved.

**Reply by mkurz** [reply]
Fixed that, see https://github.com/playframework/playframework/pull/11655

**Reply by Rapierian** [reply]
thanks!

**Reply by mkurz** [reply, marked as answer]
@Rapierian You should now be able to use evolutions with MariaDB with Play 2.8.20, which got released just now: https://github.com/playframework/playframework/releases/tag/2.8.20

**Reply by Rapierian** [reply]
Thank you!
